### PR TITLE
Rename Repo References

### PR DESCRIPTION
**Pull request by ghickman**

### CODE REVIEW

**Review by CarolineMorton — APPROVED** [review]
This looks good to me.

**Comment by StevenMaude** [comment]
While doing slightly related thing (looking for Docker image name references), I spotted some of these, so this PR saves a job :smile:

File: Justfile [hunk 1]
```diff
@@ -120,7 +120,7 @@ build-cohort-extractor:
     set -euo pipefail
 
     [[ -v CI ]] && echo "::group::Build cohort-extractor (click to view)" || echo "Build cohort-extractor"
-    docker build . -t cohort-extractor-v2
```
Comment (StevenMaude) [inline Justfile:123]: nit: There are a few other `cohort-extractor` references in the Justfile that could be updated while editing this file here.

Equally, could be a separate PR.
